templates/client.erb

```diff
@@ -1,2 +1,193 @@
+<% content_for :requires do %>
+require "json"
+require "pathname"
+
+require "google/gax"
+require "google/gax/operation"
+require "google/longrunning/operations_client"
+
+require "google/cloud/speech/v1/cloud_speech_pb"
+require "google/cloud/speech/v1/credentials"
+<% end %>
+# <%= @service.docs.leading_comments -%>
 class <%= @service.name %>Client
+  # @private
+  attr_reader :<%= @service.name.underscore %>_stub
+
+  # The default address of the service.
+  SERVICE_ADDRESS = "<%= @service.name.underscore %>.googleapis.com"
```

Comment:
This value comes from one of the annotations, but we can fix this later since it's fine to do it this way for speech.